client/cli/src/config.rs

```diff
@@ -369,6 +369,11 @@ pub trait CliConfiguration<DCV: DefaultConfigurationValues = ()>: Sized {
 		Ok(None)
 	}
 
+	/// Get maximum number of subscriptions per connection.
```

Comment:
What does `Ok(None)` mean here? That it's using the default? If yes, maybe worth mentioning in the docs.

Reply:
Yes, it's a default imp for that `CliConfiguration trait` I just did the same as the other `rpc options` but clearly a footgun for
folks like me :(